### PR TITLE
[docs] Add missing `routeId` slashes

### PR DESCRIPTION
**Pull request by brunnerh**
Implementation was changed in https://github.com/sveltejs/kit/pull/7338
Documentation was not updated everywhere.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: documentation/docs/20-core-concepts/20-load.md [hunk 1]
```diff
@@ -551,4 +551,4 @@ Note that re-running a `load` function will update the `data` prop inside the co
 
 ### Shared state
 
-In many server environments, a single instance of your app will serve multiple users. For that reason, per-request or per-user state must not be stored in shared variables outside your `load` functions, but should instead be stored in `event.locals`.
\ No newline at end of file
+In many server environments, a single instance of your app will serve multiple users. For that reason, per-request or per-user state must not be stored in shared variables outside your `load` functions, but should instead be stored in `event.locals`.
```
Comment (brunnerh) [inline documentation/docs/20-core-concepts/20-load.md:554]: (Empty line at the end was added automatically here)

**Review by gtm-nayan — APPROVED** [review]

**Comment by gtm-nayan** [comment]
Oh, the JSDoc comments for `Page` and `RequestEvent` also need to be updated in `types/index.d.ts`. @brunnerh could you get those as well in this PR?

**Comment by brunnerh** [comment]
Should be fixed.

**Comment by Rich-Harris** [comment]
thank you!

**Pull request merged**